boards/nano_rp2040_connect/src/main.rs

```diff
@@ -386,6 +398,26 @@ pub unsafe fn main() {
             adc_mux
         ));
 
+    peripherals.i2c0.init(100 * 1000);
```

Comment:
```suggestion
    peripherals.i2c0.init(100_000);
```

Nitpick, feel free to leave this as is if you'd like. I didn't expect to see two "arbitrary" numbers multiplied here and went to the driver to figure out this was the baudrate.